packages/components/src/search-list-control/index.js

```diff
@@ -109,7 +109,7 @@ export class SearchListControl extends Component {
 		const re = new RegExp( escapeRegExp( search ), 'i' );
 		this.props.debouncedSpeak( messages.updated );
 		const filteredList = list
-			.map( ( item ) => ( re.test( item.name ) ? item : false ) )
+			.map( ( item ) => ( re.test( item.name ) ? item : false)  )
```

Comment:
Oh hmm that actually looks wrong, huh. I guess there are still some teething issues with the new lint setup

Reply:
I made a comment in this file, saved it and this issue fixed itself 🤷🏻‍♀️. 

I will go ahead and commit that fix.